components/brave_ads/browser/component_updater/resource_component.cc

```diff
@@ -180,7 +180,7 @@ void ResourceComponent::OnGetManifest(const std::string& component_id,
 
     const std::string* id = resource_value.FindStringPath(kResourceIdPath);
     if (!id) {
-      VLOG(1) << *id << " resource id is missing";
+      VLOG(1) << json << " resource id is missing";
```

Comment:
nit: could we please change to `VLOG(1) << "Resource id is missing";`

Reply:
Fixed.